extensions/amp-analytics/0.1/test/test-events.js

```diff
@@ -557,10 +557,10 @@ describes.realWin('Events', {amp: 1}, env => {
 
     it('should initalize, add listeners and dispose', () => {
       expect(tracker.root).to.equal(root);
-      expect(tracker.trackers_).to.have.length(0);
+      expect(tracker.getTrackedTimerKeys()).to.have.length(0);
 
       tracker.dispose();
-      expect(tracker.trackers_).to.have.length(0);
+      expect(tracker.getTrackedTimerKeys()).to.have.length(0);
     });
 
     it('should validate timerSpec', () => {
```

Comment:
nit: Can we have two more tests
1. test the nested timer trigger event
2. test maxTimerLength with stopSpec that never stop

Reply:
Done.